app/controllers/treatments/massage_trims_controller.rb

```diff
@@ -0,0 +1,15 @@
+class Treatments::MassageTrimsController < Treatments::TreatmentsController
+
+  def new
+    @user = user
+    @treatment = @user.treatments.new
```

Comment:
So it is!  Maybe it didn't used to be.  No matter.  I think we prefer build... probably for legacy reasons.